docs/release/fundamentals/queries-and-expressions.ipynb

```diff
@@ -0,0 +1,2023 @@
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th>classification_labels</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <td>[5, 3, 2, 8, 7]</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>[0, 6, 9, 8, 1]</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>[4, 1, 9, 7, 0]</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "  classification_labels\n",
+       "0       [5, 3, 2, 8, 7]\n",
+       "1       [0, 6, 9, 8, 1]\n",
+       "2       [4, 1, 9, 7, 0]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "t.select(t.classification['labels']).head(3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2798836c-1e35-42ba-8559-550f825eeeb5",
+   "metadata": {},
+   "source": [
+    "You can also use a convenient \"attribute\" syntax for dictionary lookups:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "be58ac2e-648e-4922-be20-82b5168de813",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th>classification_labels</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <td>[5, 3, 2, 8, 7]</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>[0, 6, 9, 8, 1]</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>[4, 1, 9, 7, 0]</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "  classification_labels\n",
+       "0       [5, 3, 2, 8, 7]\n",
+       "1       [0, 6, 9, 8, 1]\n",
+       "2       [4, 1, 9, 7, 0]"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "t.select(t.classification.labels).head(3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73d8eb1c-d449-4990-b643-4e4e729fd29f",
+   "metadata": {},
+   "source": [
+    "The \"attribute\" syntax isn't fully general (it won't work for dictionary keys that are not valid Python identifiers), but it's handy when it works.\n",
+    "\n",
+    "\n",
+    "`t.classification.labels` is another Pixeltable expression; you can think of it as saying, \"do the `'labels'` lookup from every dictionary in the column `t.classification`, and return the result as a new column.\" As before, the expression by itself contains no data; it's the query that does the actual work of retrieving data. Here's what we see if we just give the expression by itself, without a query:"
```

Comment:
Maybe we should implement Expr.__repr__() (now) to print ColumnType.__str__().

I was expecting this to invoke Expr.__str__().